code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -45,25 +45,21 @@
 
 /******BURN******/
 /*Suffix: -uri*/
-/datum/reagent/medicine/C2/ichiyuri
-	name = "Ichiyuri"
-	description = "Used to treat serious burns. Prolonged exposure can cause burns to itch."
+/datum/reagent/medicine/C2/lenturi
+	name = "Lenturi"
+	description = "Used to treat serious burns. Heals better when slow"
```

Comment:
Add a period here.

If you go with my suggested change, change the entire description to "A medicine used to treat burns. It heals slower patients more than faster ones."